### DISCUSSION TITLE
docs/core/gdb-chaining-query-scan

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#73] [General]
# docs/core/gdb-chaining-query-scan

在GoFrame框架中使用Scan方法进行ORM查询的技巧，主要包括如何将查询结果转换为struct对象和struct数组。通过示例代码说明了Scan方法的用法，如单条记录转换为struct对象、多条记录转换为struct数组等，帮助用户有效地处理数据库查询结果。

https://goframe.org/docs/core/gdb-chaining-query-scan

**Comment by tkzl** [comment]
Order排序似乎有问题，

**Reply by tkzl** [reply]
m.Order(dao.User.Columns().UpdateAt,"desc") 会被解析成ORDER BY `update_at`,`desc`

**Reply by gqcn** [reply]
感谢反馈，`v2.8.0`的这块逻辑有点小调整导致有点小问题，会在`v2.8.1`解决。
